SECURITY.md

```diff
@@ -0,0 +1,29 @@
+# Security Policy
+
+The Trixi.jl development team takes security issues seriously. We appreciate
+all efforts to responsibly disclose any security issues and will make every
+effort to acknowledge contributions.
+
+
+## Supported Versions
+
+The current stable release versions following SemVer are supported with
+security updates.
```

Comment:
```suggestion
The current stable release following [semantic versioning](https://semver.org/)
is supported with security updates.
```
?

Reply:
I added a link to the Julia interpretation of SemVer instead.